### PR TITLE
Convert to docs to actual XML docs syntax

### PR DESCRIPTION
**Pull request by davkean**
Convert old doc comment syntax to actual doc comment syntax.

### CODE REVIEW

File: src/Microsoft.VisualStudio.AppDesigner/Common/Utils.vb [hunk 1]
```diff
@@ -353,12 +353,12 @@ Namespace Microsoft.VisualStudio.Editors.AppDesCommon
...
-        '@ Sets error code and error message through IVsUIShell interface
-        '@ </summary>
-        '@ <param name="hr">error code</param>
-        '@ <param name="error message">error message</param>
-        '@ <returns></returns>
+        ''' <summary>
+        ''' Sets error code and error message through IVsUIShell interface
+        ''' </summary>
+        ''' <param name="hr">error code</param>
+        ''' <param name="error message">error message</param>
```
Comment (sharwell) [inline src/Microsoft.VisualStudio.AppDesigner/Common/Utils.vb]: ❔  Does this parameter name work in VB?

File: src/Microsoft.VisualStudio.Editors/Common/ListViewComparer.vb [hunk 2]
```diff
@@ -21,9 +21,9 @@ Namespace Microsoft.VisualStudio.Editors.Common
 
         Private _sorting As SortOrder = SortOrder.Ascending
 
-        '@ <Summary>
-        '@  Which column should be used to sort the list. Start from 0
-        '@ </Summary>
+        ''' <Summary>
```
Comment (sharwell) [inline src/Microsoft.VisualStudio.Editors/Common/ListViewComparer.vb]: ❔  Does VB allow `<Summary>` (as opposed to `<summary>`)? This PR has several cases of the former.

File: src/Microsoft.VisualStudio.Editors/DesignerFramework/ToolStripImageComboBox.vb [hunk 3]
```diff
@@ -8,11 +8,11 @@ Namespace Microsoft.VisualStudio.Editors.DesignerFramework
...
-    '@ <summary>
-    '@ This class is a toolstrip item that contains a combobox which is capable
-    '@   of holding images and having a different font for each entry in it.
-    '@ </summary>
-    '@ <remarks></remarks>
+    ''' <summary>
+    ''' This class is a toolstrip item that contains a combobox which is capable
+    '''   of holding images and having a different font for each entry in it.
+    ''' </summary>
+    ''' <remarks></remarks>
```
Comment (sharwell) [inline src/Microsoft.VisualStudio.Editors/DesignerFramework/ToolStripImageComboBox.vb]: :bulb: Empty `<remarks>` element can be removed with no negative impact (occurs a few times in this PR)

Comment (davkean) [reply src/Microsoft.VisualStudio.Editors/DesignerFramework/ToolStripImageComboBox.vb]: I'm going to do this in a follow up PR, these two projects have massive swab of empty XML elements and it's a giant rabbit hole.

File: src/Microsoft.VisualStudio.AppDesigner/Common/Utils.vb (hunk 1 above)
Comment (davkean) [reply src/Microsoft.VisualStudio.AppDesigner/Common/Utils.vb]: No it doesn't, fixed.

File: src/Microsoft.VisualStudio.Editors/Common/ListViewComparer.vb (hunk 2 above)
Comment (davkean) [reply src/Microsoft.VisualStudio.Editors/Common/ListViewComparer.vb]: Yes it does. This is across the entire tree this usage, so I'd do that in one go.

Comment (davkean) [reply src/Microsoft.VisualStudio.Editors/Common/ListViewComparer.vb]: https://github.com/dotnet/project-system/pull/3658

**Comment by davkean** [comment]
@dotnet-bot retest windows_debug please

**Pull request merged**